docs/UserGuide.md

```diff
@@ -11,117 +11,237 @@
+
+> > - Year can be omitted and assumed to be the current year: <br> 
+> > E.g. 19:30 12/10 <br> 
+> > E.g. 7:30 pm 12/10 <br> 
+
+
+> Examples:
+> * **`add`**` dinner with mom `**`from`**` 19:30 02 oct 2016 `**`to`**` 20:30 02 oct 2016`
+> * **`add`**` dinner with mom `**`from`**` 19:30 `**`to`**` 20:30 02 oct 2016`
+
+> ###### Specifying deadline : `by`
+> This parameter is used to indicate the deadline of a task.<br>
+> Format: **`by`**` [datetime]`<br>
+
+> Examples:
+> * **`add`**` submit proposal `**`by`**` 23:59 02 oct 2016`
+
+> ###### Specifying location : `at`
+> This parameter is used to indicate the venue of a task.<br>
+> Format: **`at`**` [location]`<br>
+
+> Examples:
+> * **`add`**` dinner with mom `**`at`**` home`
+
+> ###### Specifying remarks : `remarks`
+> This parameter is used to add remarks for the task.<br>
+> Format: **`remarks`**` [remarks]`<br>
+
+> Examples:
+> * **`add`**` dinner with mom `**`remarks`**` buy flowers`
+
+Examples:
+* **`add`**` Prepare meeting agenda `**`by`**` 11:00 7 oct 2016`
+* **`add`**` Jimmy's wedding banquet `**`at`**` Trinity Church `**`from`**` 19:00 `**`to`**` 22:00 11 nov 2016 `
+
+### Listing all tasks : `list`
+Shows the list of all tasks sorted temporally with the most recent ones displayed first.<br>
+Format: **`list`**` [filter]`<br>
+> [filters] available: <br>
+> * today - shows the list of tasks for today's date
+> * week - shows the list of tasks for this week
+> * month - shows the list of tasks for the current month
+> * date (e.g. 12 Oct 2016) - shows the list of tasks for the specified date
+> * completed - shows a list of tasks that were mark as done.
+
+Examples:
+* **`list`**` today`
+* **`list`**` 12 Oct 2016`
+
+### Finding all tasks containing keyword in the name: `find`
+Finds tasks whose names contain any of the given keywords.<br>
+Format: **`find`**` [keywords]`
+
+> The search is case insensitive. <br>
+> the order of the keywords does not matter. <br>
+> Only the task name is searched. <br>
+> Tasks matching at least one keyword will be returned. <br>
+> More relevant tasks (i.e. those matching more keywords) are displayed first.
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `find mom`<br>
+  Returns both `dinner with mom` and `dinner with Mom`
+* `find mom dad sister`<br>
+  Returns Any task having `mom`, `dad`, or `sister`in its name
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+> ##### Optional specifiers:
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> ###### Find tasks that contain all keywords: `all`
+> This specifier demands that only tasks containing all, not just some, keywords will be returned. <br>
```

Comment:
Great work on using the feedback and coming up with a simple and intuitive solution.  👍